src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -625,7 +625,10 @@ private void DecodeWellKnownAttributeAppliedToMethod(ref DecodeWellKnownAttribut
                 }
                 else
                 {
-                    diagnostics.Add(ErrorCode.ERR_UnscopedRefAttributeUnsupportedMemberTarget, arguments.AttributeSyntaxOpt.Location);
+                    var code = !this.UseUpdatedEscapeRules
+                        ? ErrorCode.WRN_UnscopedRefAttributeOldRules
```

Comment:
>WRN_UnscopedRefAttributeOldRules [](http://example.com/codeflow?start=36&length=32)

Are we reducing an existing error to a warning here, in cases where the previous error applied regardless of which rules were used? For instance, when compiling the following with C# 10 and .NET 6:
```csharp
class C
{
    [UnscopedRef] object F() => null;
}
```